### DISCUSSION TITLE
volume control from file at transmitter before  ESPNOW

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#847] [Q&A] [answered]
As per my understanding volume class work with PCM data because it only multiply the audio data  . 

but I am sending mp3 data from file to espnow so how to change the volume at transmitter .

**Comment by pschatzmann** [comment, marked as answer]
I think this one is obvious: just send the requested volume to the destination and handle it there.

**Reply by aqildad-create** [reply]
i am facing problem in sending meta as binary container .i  have create a  issue for this too  . 

so that's why i am trying to find a solution in transmitter side .
